### PR TITLE
Context Menu / Menu: Additional Changes in data and typo's

### PR DESCRIPTION
**Pull request by nikkithelegendarypokemonster**

### CODE REVIEW

File: apps/demos/Demos/Menu/Scrolling/Vue/data.ts [hunk 1]
```diff
@@ -1,338 +1,338 @@
 const products = [
-  {
+    {
```
Comment (EugeniyKiyashko) [inline apps/demos/Demos/Menu/Scrolling/Vue/data.ts]: Why did you change formatting here? In ContextMenu data uses indent with 2 spaces

Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Menu/Scrolling/Vue/data.ts]: oh i didnt noticed this ill fix this one

Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Menu/Scrolling/Vue/data.ts]: [42b9220](https://github.com/DevExpress/DevExtreme/pull/27440/commits/42b9220d05d00d98695c0aad1ddd96f48bb18e51)

File: apps/demos/Demos/Menu/Scrolling/Vue/data.ts [hunk 2]
```diff
@@ -1,337 +1,337 @@
 const products = [
   {
-      text: 'Electronics',
```
Comment (EugeniyKiyashko) [inline apps/demos/Demos/Menu/Scrolling/Vue/data.ts:3]: Please remove excess indents in data

Comment (EugeniyKiyashko) [reply apps/demos/Demos/Menu/Scrolling/Vue/data.ts:3]: Now, all is okey

File: apps/demos/Demos/Menu/Scrolling/Angular/app/app.service.ts [hunk 3]
```diff
@@ -63,7 +63,7 @@ const products: Product[] = [
             text: 'Mobile Broadband Devices',
           },
           {
-            text: 'Cell phones Accessories',
+            text: 'Cell Phones Accessories',
```
Comment (EugeniyKiyashko) [inline apps/demos/Demos/Menu/Scrolling/Angular/app/app.service.ts]:
```suggestion
            text: 'Cell Phone Accessories',
```

Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Menu/Scrolling/Angular/app/app.service.ts]: [4d9417e](https://github.com/DevExpress/DevExtreme/pull/27440/commits/4d9417ef321d0cd4a9c0306203f5a6e3d960fb17)

[e843d1e](https://github.com/DevExpress/DevExtreme/pull/27440/commits/e843d1e876c0db0d651ce0a86ede52213efc0358)

Comment (nikkithelegendarypokemonster) [reply apps/demos/Demos/Menu/Scrolling/Angular/app/app.service.ts]: [1378c4a](https://github.com/DevExpress/DevExtreme/pull/27440/commits/1378c4a82d830dff1af8904ad68f34a9e1b4e779)

**Review by EugeniyKiyashko — DISMISSED** [review]

**Review by Zedwag — DISMISSED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]
